3_javascript/javascript1/week1/homework/javascript.js

```diff
@@ -0,0 +1,121 @@
+    dogYear = (yearFuture - dogYearOfBirth) * 7;
+    console.log('Your dog will be ' + dogYear + ' dog years old in ' + dogYearFuture);
+} else if (choice == 'human') {
+    console.log('You have chosen the word ' + choice);
+    console.log('Your dog age calculation based on human years: ');
+    dogYear = yearFuture - dogYearOfBirth;
+    console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
+} else {
+    console.log('You have not entered dog neither human. \n Please run the page again and enter one of them to perform the calculations');
+}
+console.log('\n');
+
+
+console.log('----------- Housey pricey (A house price estimator) -----------');
+
+let JuliaHouse = {
+    wide: 5, //m
+    high: 11, //m
+    deep: 8, //m
+    gardenSizeInM2: 70, // 100M2
+    housePricePaidByJulia: 1000000
+};
+
+let JuliaHousePrice;
+
+// housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300; 
+JuliaHousePrice = (JuliaHouse.wide * JuliaHouse.high * JuliaHouse.deep) * 2.5 * 1000 + (JuliaHouse.gardenSizeInM2 * 300);
+
+if (JuliaHouse.housePricePaidByJulia > JuliaHousePrice) {
+    console.log('Julia has paid more: ' + ' She has paid: ' + JuliaHouse.housePricePaidByJulia + 'and the real price is: ' + JuliaHousePrice);
+} else {
+    console.log('Julia has paid less than Peter');
+}
+
+let PeterHouse = {
+    wide: 8, //m
+    high: 10, //m
+    deep: 10, //m
+    gardenSizeInM2: 100, // M2
+    housePricePaidByPeter: 2500000
+};
+
+let PeterHousePrice;
+PeterHousePrice = (PeterHouse.wide * PeterHouse.high * PeterHouse.deep) * 2.5 * 1000 + (PeterHouse.gardenSizeInM2 * 300);
+
+if (PeterHouse.housePricePaidByPeter > PeterHousePrice) {
+    console.log('Peter has paid more than Julia. ' + ' He has paid ' + PeterHouse.housePricePaidByPeter + ' and the real price is ' + PeterHousePrice);
+} else {
+    console.log('Peter has paid less');
+}
+
+console.log('\n');
+
+/* if (JuliaHouse.housePricePaidByJulia > PeterHousePrice.housePricePaidByPeter) {
+    console.log('Julia has paid more than Peter');
+}
+else if (JuliaHouse.housePricePaidByJulia == PeterHousePrice.housePricePaidByPeter) {
+    console.log('Julia anad Peter has paid the same amount of ' + JuliaHouse.housePricePaidByJulia);
+} else {
+    console.log('Peter has paid more than Julia');
+}*/
+
+
+console.log('----------- Ez Namey (Startup name generator) Optional -----------');
+
+let firstWords = [
+    "Easy",
+    "Awesome",
+    "Glory",
+    "Magic",
+    "Green",
+    "Powerfull",
+    "Big",
+    "Shiny",
+    "Hello",
+    "Sunny"
+];
+
+let secondWords = [
+    "Corporate",
+    "Businness",
+    "Journey",
+    "Cooperation",
+    "Travel",
+    "Trading",
+    "Knowledge",
+    "Steps",
+    "Alpha",
+    "Kayak"
+];
+const randomNumber = Math.floor(Math.random() * 10);
+
+let startupName = firstWords[randomNumber] + ' ' + secondWords[randomNumber];
+console.log("The startup: " + startupName + " contains " + startupName.length + " characters");
```

Comment:
Here is if you want to counnt precise umber of characters string has, you can use for example : `(startupName.length-1)` . Because it is counting together with an empty space between words, which is not a character 😸